plugins/inputs/couchbase/couchbase.go

```diff
@@ -55,6 +58,21 @@ func (r *Couchbase) Gather(acc telegraf.Accumulator) error {
 	return nil
 }
 
+// sanitizeURI by removing information about user and/or password from string
+// it also removes schema name from URI
+func sanitizeURI(uri string) (result string, err error) {
+
+	re, err := regexp.Compile("(\\S+:\\/\\/)?(\\S+\\:\\S+@)")
```

Comment:
Good point. In this case I think I can simplify those whole thing by removing function (`MustCompile` is gonna panic anyway).